examples/pytorch/token-classification/run_ner.py

```diff
@@ -363,6 +350,31 @@ def get_label_list(labels):
             "requirement"
         )
 
+    if model.config.label2id != PretrainedConfig(num_labels=num_labels).label2id:
+        # Some have all caps in their config, some don't.
+        label_name_to_id = {k.lower(): v for k, v in model.config.label2id.items()}
```

Comment:
I think labels are all uppercased for NER so this line will be harmful (for text classification they are always lowercased coming from datasets).